openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/TestSuiteRepository.java

```diff
@@ -238,6 +307,47 @@ public RestUtil.DeleteResponse<TestSuite> deleteLogicalTestSuite(
     return new RestUtil.DeleteResponse<>(updated, changeType);
   }
 
+  private void updateTestSummaryFromBucket(JsonObject bucket, TestSummary testSummary) {
+    String key = bucket.getString("key");
+    Integer count = bucket.getJsonNumber("doc_count").intValue();
+    switch (key) {
+      case "success":
```

Comment:
we must have enum in json schema for this?

Reply:
From my understanding case value in the switch statement need to be constant known at compile time. The enum value returns a TestCaseStatus type while what we get from ES is a string. Calling `Success.value()` on the case statement throws a private access error.

There is also the lower part the Enum is `Success` while what we get from the bucket is `success` - though setting a nonNormalized field could remove this issue.

LMK if that makes sense or if you have suggestions I am not aware of.

Reply:
@TeddyCr I was expecting Status status = Status.valueOf(bucket.getString("key")) 
use that do the switch case

Reply:
Make sense. I tried to not cast it to keep it generic across aggregation